test/extended/prometheus/prometheus.go

```diff
@@ -159,13 +201,50 @@ var _ = g.Describe("[sig-instrumentation][Late] OpenShift alerting rules [apigro
 				violations.Insert(
 					fmt.Sprintf("WARNING: Alert %q is critical and has no 'runbook_url' annotation", alert.Name),
 				)
-			} else if runbook != "" {
-				// If there's a 'runbook_url' annotation, make sure it's a
-				// valid URL and that we can fetch the contents.
-				if err := helper.ValidateURL(runbook, 10*time.Second); err != nil {
+			}
+
+			return violations
+		})
+
+		if err != nil {
+			e2e.Failf(err.Error())
+		}
+	})
+
+	g.It("should have valid runbook URLs", func() {
```

Comment:
```suggestion
	g.It("should link to a valid URL if the runbook_url annotation is defined", func() {
```

Reply:
(nit) to be more specific and distinguish from the next test
```suggestion
	g.It("should link to an HTTP(S) location if the runbook_url annotation is defined", func() {
```